app/src/main/java/com/peacecorps/pcsa/MainActivity.java

```diff
@@ -213,48 +265,31 @@ public static void swapFragmentIn(FragmentActivity activity, Fragment fragment,
      */
     private void prepareListData() {
 
-        listDataHeader = new ArrayList<String>();
         listDataChild = new HashMap<String, List<String>>();
-        listDataHeader.add(getString(R.string.get_help));
-        listDataHeader.add(getString(R.string.circle_title));
-        listDataHeader.add(getString(R.string.safety_tools));
-        listDataHeader.add(getString(R.string.support_services));
-        listDataHeader.add(getString(R.string.sexual_assault_awareness));
-        listDataHeader.add(getString(R.string.policies_glossary));
-        listDataHeader.add(getString(R.string.settings));
-        listDataHeader.add(getString(R.string.user_login));
+        String[] headers = new String[]{getString(R.string.get_help),getString(R.string.circle_title),getString(R.string.safety_tools),
```

Comment:
What I suggested was instead of having them in the String.xml, have them in the array.xml as locations. So you can easily get the array and make it a list. Clear?